### PR TITLE
Update Readme with the github actin badge

### PR DESCRIPTION
**Pull request by runderwoodcr14**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/Flaconi/docker-atlantis-terragrunt.svg?branch=master)](https://travis-ci.com/Flaconi/docker-atlantis-terragrunt)
+[![Build Status](https://github.com/Flaconi/docker-atlantis-terragrunt/workflows/Build-Publish/badge.svg)](https://github.com/Flaconi/docker-atlantis-terragrunt)
```
Comment (cytopia) [inline README.md:1]:
```suggestion
[![Build Status](https://github.com/Flaconi/docker-atlantis-terragrunt/workflows/Build-Publish/badge.svg)](https://github.com/Flaconi/docker-atlantis-terragrunt/actions?query=workflow%3ABuild-Publish)
```

**Review by cytopia — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cytopia) [reply README.md:1]: This is to go directly to the actions on click

Comment (runderwoodcr14) [reply README.md:1]: Its a public repo, don't we want this?

Comment (cytopia) [reply README.md:1]: Actions are public as well anyway

Comment (cytopia) [reply README.md:1]: It's just the same behaviour we had with Travis bevor. Build badge link goes to buiild

**Review by cytopia — APPROVED** [review]
FYI @runderwoodcr14 you could have clicked the "suggestion link" to auto-apply my suggestions ;-)

**Comment by cytopia** [comment]
Screenshot 2020-12-03 18-40-19  selection

**Comment by runderwoodcr14** [comment]
I'm old fashion, prefer the old edit commit and push

**Pull request merged**